src/index.js

```diff
@@ -1,2 +1,60 @@
-/* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+const INIT_COUNT = 0;
+const INCREASE = 1;
+const BUTTON = [1, 2, 3];
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function render(count) {
+  const handleClick = () => {
+    render(count + INCREASE);
+  };
+
+  const handleClickNumberBtn = (number) => {
+    render(number);
+  };
+
+  const element = (
+    <div id="hello" className="greeting">
+      <p>hello, Olive!</p>
+      <p>
+        <button type="button" onClick={() => handleClick(count)}>
```

Comment:
매개변수로 받는 `count` 는 어디서 사용되나요?

아래 코드처럼 `onClick` 에 함수를 넘겨주는 것과 화살표 함수로 넣어주는 것은 어떤 차이가 있을까요?

```suggestion
        <button type="button" onClick={handleClick}>
```

Reply:
https://github.com/CodeSoom/react-week1-assignment-1/pull/109#issuecomment-898878660